Docs/source/running_cpp/parameters.rst

```diff
@@ -1774,7 +1774,7 @@ Lookup tables store pre-computed values for functions used by the QED modules.
     is quite low).
 
     * ``generate``: a new table is generated. This option requires Boost math library
-      (version >= 1.67) and to compile with QED_TABLE_GEN=TRUE. All
+      (version >= 1.67) and to compile with ``QED_TABLE_GEN=TRUE``. All
```

Comment:
```suggestion
      (version >= 1.66) and to compile with ``QED_TABLE_GEN=TRUE``. All
```
(see my other comment)

Reply:
Ups, PR went already in. Will put them in a follow-up PR :)